libs/user/validations.ts

```diff
@@ -105,19 +105,32 @@ export const makeTransferOptions = makeTransaction
   })
   .with("atomicHash", ["atomicTimestamp"]);
 
-export const makeTransformTransferOptions = makeTransaction
-  .append({
-    recipientNightfallAddress: Joi.string().trim().required(),
-    isOffChain: Joi.boolean().default(false),
-    providedCommitments: Joi.array().items(Joi.string()),
-    regulatorUrl: Joi.string().trim(),
-    atomicHash: Joi.string().trim(),
-    atomicTimestamp: Joi.number(),
-    salt: Joi.string().trim(),
-    inputTokens: Joi.array().items(Joi.string()),
-    outputTokens: Joi.array().items(Joi.string()),
-  })
-  .with("atomicHash", ["atomicTimestamp"]);
+export const makeTransformTransferOptions = Joi.object({
```

Comment:
Maybe we could extend an existing method...? :thinking:

Reply:
I'll review